### DISCUSSION TITLE
Expand all Disclosures or certain ones

### DISCUSSION THREAD

**Discussion by ycaoT** [twilio-labs/paste#1972] [Ideas] [answered]
Or if there is a way to do such, please share. We need a button to expand all Disclosures, so looking for a solution.

**Comment by TheSisb** [comment, marked as answer]
Hi @ycaoT,

You will need to tap into the state hook of the Disclosure. The second example in this section will show you how to do it: https://paste.twilio.design/components/disclosure#using-state-hooks

**Reply by ycaoT** [reply]
Will that be able to apply to all Disclosures? If I have multiple ones on the UI.

**Reply by TheSisb** [reply]
Here's how: https://codesandbox.io/s/toggle-all-disclosures-5bjic?file=/src/index.tsx

**Reply by kelseycavitt** [reply]
@ycaoT, does @TheSisb's link help you?
